packages/teams-js/test/public/media.spec.ts

```diff
@@ -745,19 +742,16 @@ describe('media', () => {
       });
 
       it('videoController stop function returns SdkError to callback when parent rejects message"', async () => {
```

Comment:
Can you remove the `"` after "message" in the test name? (I'm a bit surprised that doesn't cause an error actually)

Reply:
it doesn't cause an error because there is no other '"' in the string, but I'm not sure why it is there in the first place